packages/@aws-cdk-testing/framework-integ/test/aws-lambda-nodejs/test/integ-handlers/bun/_package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
cc @rix0rrr - I see that you were working with this setup in #33022. Does this setup seem reasonable to you? I need to make sure that the lockfile will be modified during bundling to avoid the origin issue this PR fixes.

Reply:
Hey @blimmer,

It seems as reasonable as anything. I honestly don't quite know what the best way is to go about updating this test, I know it gave me a lot of grief in my previous PR.

I confess I haven't dived into the possible solutions too deeply, so you have more context than I do on this right now. If you think this is the way to go, I'll happily give it a shot!

Reply:
Gotcha. I TDD'ed this, and before my change, this test was broken. So I think this is as good as anything 👍